### ISSUE TITLE
The Jade runtime should be added to the app.bundle.js

### ISSUE THREAD

**Issue by hawkerboy7** [Industrial/id-builder#26]
I made it into a separate issue.
``` coffeescript
window.jade = require('jade').runtime
```
This code is now needed to get templates to work correctly.
The jade runtime should probably be in the app.bundle.js already and therefore use the jade from id-builder instead of the user having to npm install it into their project too.

**Comment by Industrial** [comment]
I want the jade runtime to only run on the server. I only want compiled JS functions that don't need a runtime on the client. This is why id-builder just compiles all .jade files to .js files; Just do

```javascript
import myTemplate from './myTemplate';
```

in a JavaScript file that is `build/js/app.js` or a file used by it and browserify will use it like any other JS file :-)

**Comment by hawkerboy7** [comment]
I am getting an error doing so: Unexpected reserved word

**Comment by Industrial** [comment]
Sorry, that's my fault. I meant to say `src/js/app.js`, not `build/js/app.js`.

**Comment by hawkerboy7** [comment]
So this won't work using coffeescript?

**Comment by Industrial** [comment]
My example was in ES6, in coffeescript it would be

```coffee
myTemplate = require '../templates/main'
console.log myTemplate name: 'test'
```

Well crap.

I made a huge assumption: That jade doesn't need a runtime to run in the browser if you precompile the functions. I don't understand why it doesn't do this...

I've tried to use the module `jadify` instead but it doesn't work yet. I've created a ticket for a problem I have here: https://github.com/domenic/jadeify/issues/31
